app/src/main/java/org/gdg/frisbee/android/api/GdgXHubClient.java

```diff
@@ -0,0 +1,57 @@
+package org.gdg.frisbee.android.api;
+
+import android.content.Context;
+
+import com.google.gson.FieldNamingPolicy;
+
+import org.gdg.frisbee.android.BuildConfig;
+import org.gdg.frisbee.android.api.deserializer.ZuluDateTimeDeserializer;
+import org.gdg.frisbee.android.utils.Utils;
+
+import retrofit.RestAdapter;
+import retrofit.converter.GsonConverter;
+
+/**
+ * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
+ *
+ * @author Marcus Gabilheri
+ * @version 1.0
+ * @since 7/26/15.
+ */
+public final class GdgXHubClient {
+
+    // This variable is meant to be changed as pleased while debugging
+    private static final RestAdapter.LogLevel LOG_LEVEL = RestAdapter.LogLevel.NONE;
+
+    // The rest adapter that our api uses
+    private static RestAdapter restAdapter;
+
+    private static GdgXHub hubApi;
```

Comment:
Probably not since we already have the instance in the Application class. I'll modify that.